detect_secrets/core/baseline.py

```diff
@@ -12,7 +12,7 @@ def initialize(
     plugins,
     exclude_files_regex=None,
     exclude_lines_regex=None,
-    path='.',
```

Comment:
Generally speaking, it's a bad idea to have a list as a default parameter, due to the following reason:

```
def foobar(L=['a']):
    L.append('b')
    print(L)

foobar()    # ['a', 'b']
foobar()    # ['a', 'b', 'b']
```

While we may not be appending to `path` here, we're introducing a "gotcha" that may come to bite us later on. A better way of doing this is:

```
def foobar(L=None):
    if not L:
        L = []
```

And since we're doing this, we can also do:

```
def foobar(L='.'):
    if not isinstance(L, list):
        L = [L]
```

which provides a nicer API for us to use (rather than always calling the function with a list of length 1).

Reply:
Thanks for pointing this :) Tried to solve It on the last update, looks better now?